DFE-Digital/school-account
# Build preliminary frontend for MVC skeleton

## Summary of changes
- Added styling to various pages, approaching with a mobile first design (using the Department for Education (DfE) Design System for information/inspiration)
- Built header based on design on DfE site
- Ensured that pages on the application print in a sensible manner (logo/service name printing in B&W, removing unnecessary elements)
- Styling for the Contentful cards has been added (thanks to @kruncher)

## How to test
- Build the solution and navigate around the site. Ensure that you've built and are serving the styling assets (`npm i` then `npm run build` in the `Dfe.SchoolAccount.Web` directory).

## Accessibility features
- The logo and service name have a focus state that's been built through CSS - this has also been built 'manually' on the DfE's official site as opposed to using focus states that come with GOVUK Frontend
- Given that we cannot add the DfE Frontend directly to our application, efforts have been made to recreate the DfE header faithfully but may not be perfectly 1:1

## Notes
- Links in the footer (Cookies, Accessibility etc.) haven't been created and thus aren't currently working aside from the external Crown copyright hyperlink
